content/guides/migrating.md

```diff
@@ -368,7 +368,7 @@ module.exports = {
   module: { 
     rules: [{ 
       test: /\.tsx?$/,
-      loader: 'ts-loader'
+      use: 'ts-loader'
```

Comment:
`use` and `options` cannot be used together. If I remember right, validation should complain. A little confusing, I know.